packages/thunderbird.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Thunderbird'
+$category = 'Email'
```

Comment:
The categories are fixed and there is no Email category. See discussion in https://github.com/mandiant/VM-Packages/issues/38

@mr-tz should we add a category check to the linter?

Reply:
Isn't there one? Checks didn't run here yet.

I'm also unsure if we should add this here.

Reply:
There's not category check, so we could add that. Still confusing that no Actions run here.

Reply:
This is what you meant https://github.com/mandiant/VM-Packages/issues/270 😅